packages/cli/src/lib/bundler/transforms.ts

```diff
@@ -35,7 +35,12 @@ export const transforms = (options: TransformOptions): Transforms => {
   const extraTransforms = isDev ? ['react-hot-loader'] : [];
 
   const transformExcludeCondition = {
-    and: [/node_modules/, { not: externalTransforms }],
+    or: [
+      // This makes sure we don't transform node_modules inside any of the local monorepo packages
+      /node_modules.*node_modules/,
```

Comment:
What's an example of what a path like this would look like? Is this because of symlinks leading to another package, which in turn also has a node_modules?

Reply:
For example `<repo-root>/node_modules/@backstage/core/node_modules/react`, and yup. With symlinks followed the path would be seen as `<repo-root>/packages/core/node_modules/react`.

The thing that broke is that we explicitly include `<repo-root>/node_modules/@backstage/core` in transforms so that we can disable following symlinks during resolution, but that also included the `node_modules` inside it.